tests/scale_test/utils/utils.py

```diff
@@ -80,15 +87,18 @@ def prepare_notification(
     Args:
         bundle_uuid (str): A Blue Box bundle uuid.
         bundle_version (str): A Blue Box bundle version.
-        es_query_path (str): The path to the ES query json file which is used for making subscription in BlueBox.
         subscription_id (str): A valid Lira subscription id in Blue Box.
+        workflow_name (str): The name of the workflow to start.
+        es_query_path (str): The path to the ES query json file which is used for making subscription in BlueBox.
         label (str): A label to be added to the notification, which will then be added to the workflow started by Lira.
         transaction_id (str): A valid transaction id.
 
     Returns:
         notification (dict): A dict of valid notification content.
 
     """
+    if not es_query_path:
+        es_query_path = SUBSCRIPTION_QUERIES[workflow_name]
```

Comment:
If the workflow isn't in the list, and `es_query_path` isn't provided, should this raise an error?

Reply:
Good point!